method-scripts-berterniereport.xml

```diff
@@ -14,12 +14,12 @@
   </es:transform>
   <es:transform name="BertErnieReportInBand" target="soap:Body/{$query}/{$request}">
     <!--
-    This applies to only the report method, although as the poriginal author I do know
+    This applies to only the report method, although as the original author I do know
     that the first part of the test would make transform safe to be shared with all
     service methods except OscarSearch.
     -->
     <es:if test="$internal_user=0 and $test_user=1">
-      <es:append-to-value target="Row/Options/IsOnline" value="&apos;1&apos;"/>
+      <es:set-value target="Row/Options/IsOnline" value="&apos;1&apos;"/>
```

Comment:
I keep seeing examples of escaped single quotes.  In attributes I think that is only necessary if the attribute value itself is single quoted.  I find un-escaped quotes easier to read.  Is this a preference?

Reply:
Or is there some problem when they are not escaped?

Reply:
If I remember correctly, the evaluation of the unquoted value did not yield the intended results. Otherwise I would have preferred unquoted. We can retry it, but this is an accurate representation of what we've tested.

Reply:
BTW, this specific line's purpose is to ask the roxie to not generate a deltabase dataset. I know, it's obvious. When logging control is available, this line will be replaced with a "disable deltabase" statement so the ESP won't have to rely on the roxie to fulfill its requirement.

Reply:
In my unit tests I don't escape single quotes.

Reply:
Just speculating, but I think there's a difference between your compiled unit tests and a script loaded from dali. I didn't have success with unquoted, and quoted only worked when escaped.

It's a worthwhile discussion to understand how scripts can be improved in general, but is it a necessary discussion for this pull request?

Reply:
It's possible that unescaped quotes would be accepted when publishing to dali, but as I discovered recently what goes into dali (by way of the esdl config service and whichever front end is used to publish) isn't what comes out of dali. When I wrapped a script in CDATA, what came back was escaped XML with no CDATA. The configuration from which this sample was derived was probably a configuration extracted from dali.